cosinnus/api_frontend/views/navigation.py

```diff
@@ -721,24 +743,26 @@ def get(self, request):
 
         # profile page
         profile_menu_items = [
-            MenuItem(_('My Profile'), reverse('cosinnus:profile-detail'), 'fa-circle-user'),
+            MenuItem(_('My Profile'), reverse('cosinnus:profile-detail'), 'fa-circle-user', id='Profile'),
         ]
         if settings.COSINNUS_V3_FRONTEND_ENABLED:
             profile_menu_items.append(
-                MenuItem(_('Set up my Profile'), reverse('cosinnus:v3-frontend-setup-profile'), 'fa-pen'),
+                MenuItem(_('Set up my Profile'), reverse('cosinnus:v3-frontend-setup-profile'), 'fa-pen',
+                         id='SetupProfile'),
             )
         profile_menu_items.extend([
-            MenuItem(_('Edit my Profile'), reverse('cosinnus:profile-edit'), 'fa-gear'),
-            MenuItem(_('Notification Preferences'), reverse('cosinnus:notifications'), 'fa-envelope'),
+            MenuItem(_('Edit my Profile'), reverse('cosinnus:profile-edit'), 'fa-gear', id='EditProfile'),
+            MenuItem(_('Notification Preferences'), reverse('cosinnus:notifications'), 'fa-envelope',
+                     id='NotificationPreferences'),
 
         ])
         profile_menu.extend(profile_menu_items)
 
         # language
         if not settings.COSINNUS_LANGUAGE_SELECT_DISABLED:
-            language_item = MenuItem(_('Change Language'), None, 'fa-language')
+            language_item = MenuItem(_('Change Language'), None, 'fa-language', id='ChangeLanguage')
             language_subitems = [
-                MenuItem(language, reverse('cosinnus:switch-language', kwargs={'language': code}))
+                MenuItem(language, reverse('cosinnus:switch-language', kwargs={'language': code}), id=code)
```

Comment:
This makes the sub-menu-items for the language change menu have ids like "de", "en". Should they maybe get a unique prefix like "ChangeLanguageItem-de" or similar?